Examples/SampleApp/_flowconfig

```diff
@@ -15,6 +15,9 @@
 .*/node_modules/react-tools/src/core/ReactInstanceHandles.js
 .*/node_modules/react-tools/src/event/EventPropagators.js
 
+# Ignore flow-bin
+.*/node_modules/flow-bin/.*
```

Comment:
We currently package the Flow binary with some examples. Maybe we should stop doing that...